### PR TITLE
BL-481 <unit tests for registration features>

### PR DESCRIPTION
**Pull request by kevkenshi-max**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Loom Video

Please paste a link here of a quick loom video walking through what you did in your PR .

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by kevkenshi-max** [comment]
https://www.loom.com/share/15010644ab974e8ab57459f7e941e0ca

File: src/__tests__/Registration.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducers from '../redux/reducers/index';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+// import  RegistrationPage  from '../components/pages/Registration/RegisterStep1';
```
Comment (dirkknibbe) [inline src/__tests__/Registration.test.js:9]: Hey guys, the loom video was pretty clean and explanatory, why is this line commented out though?

File: src/__tests__/Registration.test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        isAuthenticated: true,
+      },
+      authService: {},
+    };
+  },
+}));
+
+describe('<Instructor Welcome />', () => {
+  test('it renders the instructorWelcome component', () => {
+    const { getByText } = render(
```
Comment (dirkknibbe) [inline src/__tests__/Registration.test.js:27]: the tests look simple and seem to work great, why does getBytext have this linter warning?

**Review by dirkknibbe — APPROVED** [review]

**Review by AnthonyLayne — APPROVED** [review]
The tests look good, but I would get rid of all the commented out code. It also looks like 'getByText' isn't being used, I don't know if you have plans for it to be used in the future, if so, then maybe add a comment to indicate what it will be used for. Other than that it looks good.

**Review by EmmS21 — APPROVED** [review]
Tests look well written, may want to either remove or uncomment the commended out code, if you don't need it, maybe don't keep it

**Review by Rlauckern — APPROVED** [review]
Tests look great and so does the loom video. It seems to have been updated with the commented out lines removed. Nice.

**Review by ivanovicuros — APPROVED** [review]

**Pull request merged**